k8s/pvc.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    app: palworld-server
+  name: palworld-server-datadir
+  namespace: default
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: local-path
```

Comment:
storageClassName and volumeMode shouldn't be specified with `local-path`. The k8s server has default storage providers that can automatically provision it. 

(I use local-path too with k3s / k0s too.. haha).

volumeMode

Reply:
good point, I'm running longhorn in my homelab so local-path would kill my usage.